tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -517,6 +585,60 @@ def test_llmobs_anthropic_chat_model(self, langchain_anthropic, mock_llmobs_span
         assert mock_llmobs_span_writer.enqueue.call_count == 1
         _assert_expected_llmobs_llm_span(span, mock_llmobs_span_writer, input_role="user")
 
+    @pytest.mark.skipif(sys.version_info < (3, 10, 0), reason="Requires unnecessary cassette file for Python 3.9")
+    def test_llmobs_embedding_query(self, langchain_community, mock_llmobs_span_writer, mock_tracer):
+        if langchain_community is None:
+            pytest.skip("langchain_community not installed which is required for this test.")
+        embedding_model = langchain_community.embeddings.OpenAIEmbeddings()
+        trace = self._embed_query(
```

Comment:
```suggestion
        with mock.patch("langchain_openai.OpenAIEmbeddings._get_len_safe_embeddings", return_value=[0.0] * 1536):
            trace = self._embed_query(
```
There is a langchain internal method that relies on an external dependency that we need to mock.

Reply:
Question : How would I be able to know about this myself ? 😅 Wouldn't the test surface the dependency ?